packages/jsii-diff/bin/jsii-diff.ts

```diff
@@ -87,24 +88,24 @@ const NPM_REGEX = /^npm:(\/\/)?/;
  *
  * Supports downloading from NPM as well as from file or directory.
  */
-async function loadAssembly(requested: string): Promise<LoadAssemblyResult> {
+async function loadAssembly(requested: string, validate: boolean): Promise<LoadAssemblyResult> {
```

Comment:
Would we accept the boolean as part of an options hash? Then you can just pass `args as any` in...